charts/testkube-cloud-api/templates/deployment.yaml

```diff
@@ -326,10 +326,16 @@ spec:
               value: "{{ .Values.api.features.bootstrapConfig.enabled}}"
             - name: "BOOTSTRAP_CONFIG_PATH"
               value: "/app/config/config"
+            - name: "TESTKUBE_DEFAULT_AGENT_TOKEN"
+              valueFrom:
+                secretKeyRef:
+                  name: testkube-default-agent-token
+                  key: agent-token
+                  optional: true
             {{- end }}
             {{- range $k, $v := .Values.additionalEnv }}
             - name: {{ $k }}
-              value: "{{ $v }}"
+              value: {{ tpl ($v | toString) $ | quote }}
```

Comment:
What's your opinion on using `tpl` more liberally so we can create dynamic parameters? For example, `OAUTH_TOKEN_URL: http://testkube-enterprise-dex.{{.Release.Namespace}}.svc.cluster.local:5556/token`. Here it allows us to install Testkube on any namespace.

Reply:
It is a good use of it.